src/lib/models/queryBuilder.ts

```
@@ -43,13 +43,13 @@ export class QueryBuilder {
 		return this;
 	}
 
-	public deploymentVar(key: string, value: string | number | boolean, enforce: boolean = true) {
+	public deploymentVar(key: string, value: string | number | boolean | string[], enforce: boolean = true) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)